addons/mail/static/src/components/partner_im_status_icon/partner_im_status_icon.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.PartnerImStatusIcon" owl="1">
```

Comment:
Not a super big deal for this PR right now:
Would adapt this component so that the prop and getter is no longer "partner", but a "person", so that it covers both `Partner` and `Guest` as it is now.